src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static Random random = new Random();
+    private static ColorSupplier colorSupplier = new ColorSupplier();
+    private static final int NUMBER = 101;
```

Comment:
```suggestion
    private static final int MAX_BALL_NUMBER = 101;
```

lets make constant names more informative